src/ValidationException.php

```diff
@@ -44,12 +44,22 @@ public function __construct(MessageBag $errors)
     }
 
     /**
+     * Get the validation errors.
+     *
+     * @return \Illuminate\Support\MessageBag
+     */
+    public function getErrors()
```

Comment:
I didn't call it getErrors on purpose to be consistent with laravel's contracts. getErrors is an alias so the original method should always be preferred.